core/client/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -262,7 +262,7 @@ public FileInStream openFile(AlluxioURI path, OpenFileOptions options)
       throw new FileNotFoundException(
           ExceptionMessage.CANNOT_READ_DIRECTORY.getMessage(status.getName()));
     }
-    return FileInStream.create(status, options.toInStreamOptions());
+    return FileInStream.create(status, options.toInStreamOptions(), FileSystemContext.INSTANCE);
```

Comment:
oh, I didn't realize I'd already dependency injected this class, switched to `mContext` 👍